release notes/v0.46.0.md

```diff
@@ -0,0 +1,95 @@
+
+- (_optional_) `<highlight of breaking changes>`
+- `<Summary of new features>` (_one or multiple bullets_)
+
+
+## Breaking changes
+
+- `#pr`, `<small_break_1>`
+- `#pr`, `<small_break_2>`
+
+### (_optional h3_) `<big_breaking_change>` `#pr`
+
+## New features
+
+### Loki log output sending additional headers [#3227](https://github.com/grafana/k6/pull/3227)
+
+k6 has been able to send logs to loki for nearly 3 years since [v0.28.0](https://github.com/grafana/k6/releases/tag/v0.28.0) but didn't support any way to authenticate.
+
+Now it can be configured to send additional headers on every request.
+
+This is being configured with the new `header` config option similar to `label`:
+
+```
+k6 --log-output=loki=http://example.org,header.X-My-Header=123,header.Authorization=mytoken ...
+```
+
+Will now send the header `X-My-Header` with value `123` and `Authorization` with `mytoken`.
+
+Thanks to @federicotdn for doign the changes
+
+### Cloud Traces [#3100](https://github.com/grafana/k6/pull/3100), [#3202](https://github.com/grafana/k6/pull/3202)
+
+This release supports the new integration between k6 and Tempo in the cloud. Grafana Cloud k6 and Grafana Cloud Traces customers will be able to start their traces in k6 using the existing `k6/experimental/tracing` module to enrich their test run analysis page with metrics and aggregations from tracing data.
+
+The new cloud traces will work "out of the box" for `k6 cloud` runs. In case of `k6 run` execution, the `K6_CLOUD_TRACES_ENABLED` environment variable has to be set to `true`.
+
+### `<big_feature_1>` `#pr`
+
+_what, why, and what this means for the user_
+
+### `<big_feature_n>` `#pr`
+
+_what, why, and what this means for the user_
+
+### UX improvements and enhancements
+
+_Format as `<number> <present_verb> <object>. <credit>`_:
+- _`#999` Gives terminal output prettier printing. Thanks to `@person` for the help!_
+
+- TODO: [#3157](https://github.com/grafana/pull/3157) and [#3172](https://github.com/grafana/pull/3172) Add a `MaxTimeSeriesInBatch` configuration option.
+- [#3159](https://github.com/grafana/pull/3159) Adds per-vu TLS configuration to `grpc.Client` connect method's parameters. Thanks @chrismoran-mica 🙇‍♂️.
+- [#3176](https://github.com/grafana/pull/3176) Adds a `js/promises` package, which enables extension developers to easily create promises that will be dispatched to the eventloop using the `New` function.
+- [#3181](https://github.com/grafana/pull/3181) Adds a `RunOnEventLoop` method to the `modulestest.Runtime` type, which allows extensions developers to run code on the event loop from their tests.
+
+## Bug fixes
+
+- [#3163](https://github.com/grafana/pull/3163) Addresses a bug in our gRPC example.
+- [#3178](https://github.com/grafana/pull/3178) Fixes the registration of nested protobuf messages in the gRPC module.
+- [#3170](https://github.com/grafana/pull/3170) Upgrades the version of gRPC k6 depends on to address a security vulnerability ([CVE-2023-32731]).
```

Comment:
The #3163 and #3170 are rather maintenance or internal improvements than bugs. So I'd suggest moving them to that section.

The #3178 was reported & fixed by the external contributor, let's add this.

```suggestion
- [#3178](https://github.com/grafana/k6/pull/3178), [xk6-grpc#23](https://github.com/grafana/xk6-grpc/pull/23) Fixes the registration of nested protobuf messages in the gRPC module. Thanks @thiagodpf for reporting and actually fixing the bug!
```